proto/libp2p.proto

```diff
@@ -87,11 +87,17 @@ message NewPeer {
     bytes peer_id = 1;
 }
 
+message ResultMessage {
+    repeated bytes message = 3;
```

Comment:
Is this a raw binary or is it just a string?

Reply:
A string when errors happen, and a raw binary on most commands that return a result. Do you think we should make that distinction?

Reply:
Ah, makes sense. No worries.